packages/sdk/tests/e2e/dataProtector/fetchGrantedAccess.test.ts

```diff
@@ -1,20 +1,20 @@
 import { describe, it, beforeEach, expect } from '@jest/globals';
 import { HDNodeWallet, Wallet } from 'ethers';
-import { IExecDataProtector, getWeb3Provider } from '../../src/index.js';
-import { ValidationError } from '../../src/utils/errors.js';
+import { DataProtector, getWeb3Provider } from '../../../src/index.js';
+import { ValidationError } from '../../../src/utils/errors.js';
 import {
   MAX_EXPECTED_BLOCKTIME,
   MAX_EXPECTED_WEB2_SERVICES_TIME,
   deployRandomApp,
   getRandomAddress,
-} from '../test-utils.js';
+} from '../../test-utils.js';
 
 describe('dataProtector.fetchGrantedAccess()', () => {
-  let dataProtector: IExecDataProtector;
+  let dataProtector: DataProtector;
   let wallet: HDNodeWallet;
   beforeEach(async () => {
     wallet = Wallet.createRandom();
-    dataProtector = new IExecDataProtector(getWeb3Provider(wallet.privateKey));
```

Comment:
Changing how to instantiate the SDK, pretty big change.
Let's add some item in the Changelog. One quick line at first, just not to forget.

Reply:
yeah good iear

Reply:
Commit: 09c39858b08dd7c53e485a3c693efa77513778e2